### PR TITLE
Add to the caterpillar dialog.

### PR DESCRIPTION
**Pull request by symedley**
@casey-collab , please review my changes to the caterpillar dialog

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1750)

### CODE REVIEW

File: english/coffee/drink-tea/alice/shriek/shriek.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+You shriek at the sight of the green, bulbous, talking thing and run
```
Comment (casey-collab) [inline english/coffee/drink-tea/alice/shriek/shriek.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

File: english/coffee/drink-tea/alice/talk/talk.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+It's actually a caterpillar, you realize after a split second of revulsion.
```
Comment (casey-collab) [inline english/coffee/drink-tea/alice/talk/talk.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by symedley** [comment]
@casey-collab I've added the links between files that you suggested.

**Comment by casey-collab** [comment]
Looks great! :dog:

**Pull request merged**